contracts/package.json

```diff
@@ -42,8 +42,8 @@
     "typescript": "^4.9.3"
   },
   "dependencies": {
-    "@openzeppelin/contracts": "^4.8.3",
-    "@openzeppelin/contracts-upgradeable": "^4.8.3",
+    "@openzeppelin/contracts": "^4.9.3",
```

Comment:
Updated to newer openzepplin with the AccessControlDefaultAdminRulesUpgradeable extension.